xmtp_mls/src/client.rs

```diff
@@ -106,9 +108,9 @@ pub enum MessageProcessingError {
     #[error("storage error: {0}")]
     Storage(#[from] crate::storage::StorageError),
     #[error("tls deserialization: {0}")]
-    TlsDeserialization(#[from] tls_codec::Error),
+    TlsDeserialization(#[from] openmls::prelude::Error),
```

Comment:
If this is now a more general MLS error type, maybe we need to rename our mapped type. I'm assuming `openmls::prelude::Error` now includes more things than `TlsDeserialization` errors, although correct me if I'm wrong.